src/index.tsx

```diff
@@ -1,10 +1,17 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 
-import './index.css';
+import './css/shared-stylesheet.css';
 import App from './components/App';
 import * as serviceWorker from './serviceWorker';
 
+const org = process.env.REACT_APP_THEME;
```

Comment:
Can this be removed now? I see the same lines still in the theme.ts file

Reply:
Unfortunately not as this is to import the additional css file. In theme.ts, that is used to choose which theme to use.